### PR TITLE
Update compiler-plugins.md

### PR DESCRIPTION
**Pull request by DipanshKhandelwal**

### CODE REVIEW

File: pages/docs/reference/compiler-plugins.md [hunk 1]
```diff
@@ -117,9 +117,9 @@ plugins {
 The Maven example is similar to the one above.
 
 The plugin specifies the following annotations: 
-[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html), 
-[`@Async`](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/scheduling/annotation/Async.html), 
-[`@Transactional`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html), 
+[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html);
```
Comment (h0tk3y) [inline pages/docs/reference/compiler-plugins.md]: There's no need in semicolons where there's a simple comma-separated list of declarations (see how this list is [currently rendered](https://kotlinlang.org/docs/reference/compiler-plugins.html#kotlin-spring-compiler-plugin)).

Comment (DipanshKhandelwal) [reply pages/docs/reference/compiler-plugins.md]: Ya, i got that. Thanks.

File: pages/docs/reference/compiler-plugins.md [hunk 2]
```diff
@@ -117,9 +117,9 @@ plugins {
 The Maven example is similar to the one above.
 
 The plugin specifies the following annotations: 
-[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html), 
-[`@Async`](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/scheduling/annotation/Async.html), 
-[`@Transactional`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html), 
+[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html)
```
Comment (h0tk3y) [inline pages/docs/reference/compiler-plugins.md]: I mean, it should be a comma-separated list, as it was originally. It is not vertical, and without commas it looks mis-formatted:

image

**Comment by DipanshKhandelwal** [comment]
OK, thanks for the review.

**Comment by DipanshKhandelwal** [comment]
@h0tk3y , is it alright now?

**Pull request merged**